src/lib/viewers/doc/__tests__/DocFirstPreloader-test.js

```diff
@@ -826,4 +826,99 @@ describe('/lib/viewers/doc/DocFirstPreloader', () => {
             expect(document.getElementsByClassName).toHaveBeenCalledWith('bcpr-PreviewMask');
         });
     });
+
+    describe('processAdditionalPages()', () => {
+        beforeEach(() => {
+            // Set up preloader instance
+            preloader.pdfData = { numPages: 5 };
+            preloader.imageDimensions = { width: 123, height: 456 };
```

Comment:
Can we make the width and height constants so we can use them for test validation further down?